src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,63 @@
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedHash=Hash
+approvedHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
+approvedExpandCode=Code
+approvedLanguage=Language
+approvedScriptLength=Length
+approvedScriptLength_tooltip=Length of the script in characters
+usageCount=# of uses
+usageCount_tooltip=Usage since the introduction of metadata
+lastTimeUsed=Date of last use
+wasPreapproved_header_tooltip=The presence of a green tick means the script was approved during the configuration. \
+  Otherwise it was created by a user without the permission to run scripts and was then approved using this page.
+lastApprovalTime=Date of the last approval
+lastApprovalTime_tooltip=A single script can be approved multiple times. It occurs every time the related configuration is saved.
+lastKnownApproverLogin=Last approver
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script does not provide the user information.
+noRequestApprovalTimeSinceMetadata=N/A
+noRequestApprovalTimeSinceMetadata_tooltip=It was requested before the introduction of metadata.
+noContextItemSinceMetadata=N/A
+noContextItemSinceMetadata_tooltip=The context item was not provided by the code asking for the approval.
+
+emptyMetadata=There is no metadata for this approval
+emptyMetadata_tooltip=If the script is used after the metadata introduction, some metadata will be displayed here. \
+  It means that an unused approval will not have metadata and could be reasonably revoked after a certain period of time.
+noScriptCodeSinceMetadata=N/A
+noScriptCodeSinceMetadata_tooltip=The approval was given while the metadata were not gathered.
+approvedContextUser=Requester
+approvedContextItem=Context
+wasPreapproved_tooltip=The script content was approved directly during configuration, meaning it comes from a user with the permission to run scripts.
+wasPreapproved_alt=Preapproved
+noUsageCountSinceMetadata=N/A
+noUsageCountSinceMetadata_tooltip=It was not used since the introduction of metadata.
+notUsedSinceMetadata=N/A
+notUsedSinceMetadata_tooltip=It was not used since the introduction of metadata.
+notApprovedSinceMetadata=N/A
+notApprovedSinceMetadata_tooltip=It was not approved since the introduction of metadata.
+noKnownApproverSinceMetadata=N/A
+noKnownApproverSinceMetadata_tooltip=It was not approved since the introduction of metadata.
+
+revokeApproval=Revoke all selected
```

Comment:
I would prefer that the buttons read "Revoke/Approve selected", skipping the "all". Thoughts of a native speaker @timja ?

Reply:
Without seeing the full context I would lean towards one of:

* Revoke
* Remove
* Remove selected

Revoke isn't really a call to action text you would normally see in a UI

Reply:
Pictures wanted

Reply:
Screenshots already included in the PR description, the second one contains this button :)